docs/user/web_data.rst

```diff
@@ -396,37 +396,37 @@ Appendix A: GeoServer Jars
 +------------------------------------------------------------+------------+
 | GeographicLib-Java-1.44.jar                                | 31693      |
 +------------------------------------------------------------+------------+
-| geomesa-accumulo-datastore-1.2.0-SNAPSHOT.jar              | 2338358    |
+| geomesa-accumulo-datastore_2.11-1.3.0-m0.jar               | 3059818    |
```

Comment:
i initially added this back when people struggled to install some of these things as a 'look this exact configuration works', but maybe we can remove it now. As @cne1x points out we should have gone back and at least put the locationtech release file sizes.